### PR TITLE
Test standby corrupt md

### PR DESCRIPTION
**Pull request by mmichal10**

### CODE REVIEW

File: test/functional/tests/fault_injection/test_fault_injection_standby.py [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+        core_device = TestRun.disks["core"]
+        core_device.create_partitions([Size(500, Unit.MebiByte)])
+        core_device = core_device.partitions[0]
+
+    with TestRun.step("Prepare metadata dump"):
+        cache_id = 1
+        cls = CacheLineSize.LINE_32KiB
+        md_dump = prepare_md_dump(cache_device, core_device, cls, cache_id)
+
+    bs = Size(1, Unit.Blocks512)
```
[the remark this answers is not in the document]
Comment (mmichal10) [reply test/functional/tests/fault_injection/test_fault_injection_standby.py]: done

File: test/functional/tests/fault_injection/test_fault_injection_standby.py [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+from test_utils.filesystem.file import File
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.nand, DiskType.optane]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_activate_corrupted():
+    """
+    title: Activate cache instance on corrupted metadata
+    description: |
+      Initialize standby cache, populate it with corrupted metadata and try to activate.
```
[the remark this answers is not in the document]
Comment (mmichal10) [reply test/functional/tests/fault_injection/test_fault_injection_standby.py]: done

File: test/functional/api/cas/dmesg.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+from test_utils.size import Size, Unit
+
+
+def get_metadata_size(dmesg):
+    for s in dmesg.split("\n"):
+        if "Metadata capacity:" in s:
+            size = re.search("[0-9]* MiB", s).group()
+            return Size(int(re.search("[0-9]*", size).group()), Unit.MebiByte)
+
+    raise ValueError("Can't find the metadata size in the privded dmesg output")
```
[the remark this answers is not in the document]
Comment (mmichal10) [reply test/functional/api/cas/dmesg.py:17]: Two blank lines between functions are required by pep8

File: test/functional/api/cas/dmesg.py [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+
+def _get_metadata_info(dmesg, section_name):
+    for s in dmesg.split("\n"):
+        if section_name in s:
+            size, unit = re.search("[0-9]* (B|kiB)", s).group().split()
+            unit = Unit.KibiByte if unit == "kiB" else Unit.Byte
+            return Size(int(re.search("[0-9]*", size).group()), unit)
+
+    raise ValueError(f'"{section_name}" entry doesn\'t exist in the given dmesg output')
```
[the remark this answers is not in the document]
Comment (mmichal10) [reply test/functional/api/cas/dmesg.py:27]: Two blank lines between functions are required by pep8

**Review by pdebski21 — APPROVED** [review]

**Review by karolinavelkaja — APPROVED** [review]

**Pull request merged**